lib/ts/fetch.ts

```diff
@@ -386,23 +393,24 @@ export async function onUnauthorisedResponse(
                     // means that some other process has already called this API and succeeded. so we need to call it again
                     return { result: "RETRY" };
                 }
```

Comment:
There are places where we check the value of id refresh token to know if a session exists (instead of calling the doesSessionExist), so that needs to be changed to use the front token and the lastRefreshAttempt time stamp